WooCommerce/Classes/ViewRelated/Products/Add Product/FirstProductCreatedView.swift

```diff
@@ -0,0 +1,103 @@
+import ConfettiSwiftUI
+import SwiftUI
+
+final class FirstProductCreatedHostingController: UIHostingController<FirstProductCreatedView> {
+    init(productURL: URL) {
+        super.init(rootView: FirstProductCreatedView())
+        rootView.onSharingProduct = { [weak self] in
+            guard let self else { return }
+            SharingHelper.shareURL(url: productURL, from: self.view, in: self)
+            ServiceLocator.analytics.track(.firstCreatedProductShareTapped)
+        }
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTransparentNavigationBar()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: Localization.cancel, style: .plain, target: self, action: #selector(dismissView))
+        ServiceLocator.analytics.track(.firstCreatedProductShown)
+    }
+
+    @objc
+    private func dismissView() {
+        dismiss(animated: true)
+    }
+}
+
+private extension FirstProductCreatedHostingController {
+    enum Localization {
+        static let cancel = NSLocalizedString("Cancel", comment: "Button to dismiss the first created product screen")
```

Comment:
Super nit: IMO, "Cancel" might make it challenging to understand whether sharing will be cancelled by tapping this button. (After initiating share) WDYT about using "Close"?

Reply:
Updated the button title to "Dismiss" in 022a58fce196ca3b0372b48ede7c5cb4d9e38dd4 👍